include/tvm/relay/expr.h

```diff
@@ -267,7 +267,7 @@ class CallNode : public ExprNode {
    *
    * \endcode
    */
-  tvm::Array<Type> type_args;
+  mutable tvm::Array<Type> type_args;
```

Comment:
It may be better to use const cast selectively in the implementation rather then marking this field as mutable.

Reply:
I agree using const cast selectively is a better approach